src/code/audio_seqplayer.c

```diff
@@ -1322,20 +1320,19 @@ void Audio_SequenceChannelProcessScript(SequenceChannel* channel) {
                         channel->unk_22 = ((u16*)(channel->dynTable))[scriptState->value];
                         break;
                     case 0xB6:
-                        data = (*channel->dynTable)[0];
                         scriptState->value = (*channel->dynTable)[0][scriptState->value];
                         break;
                     case 0xB7:
-                        channel->unk_22 = (parameters[0] == 0) ? gAudioContext.audioRandom // odd load here
+                        channel->unk_22 = (parameters[0] == 0) ? gAudioContext.audioRandom & 0xffff
```

Comment:
```suggestion
                        channel->unk_22 = (parameters[0] == 0) ? gAudioContext.audioRandom & 0xFFFF
```
and other 0xffff too